app/helpers/commonwealth_vlr_engine/catalog_helper_behavior.rb

```diff
@@ -23,6 +23,18 @@ def has_playable_audio?(files_hash)
       files_hash[:audio].all? { |a| a['attachments_ss']['audio_access'].present? }
     end
 
+    def has_pdf_files?(files_hash)
+      return false if files_hash[:document].blank?
+
+      files_hash[:document].any? { |a| a['attachments_ss']['document_access'].present? }
+    end
+
+    def book_reader?(document, files_hash)
+      return false if files_hash[:image].blank?
+
+      has_searchable_text?(document) || files_hash[:image].length > 7
```

Comment:
why is it 7 just out of curiosity? any chance we can set this as a constant of make configurable?

Reply:
When there are more than 7 images the UX for the multi-image viewer starts to degrade.

Added a constant in 4db414c.